incubator/exporter-otlp/src/main/java/io/aklivity/zilla/runtime/exporter/otlp/internal/OltpExporterHandler.java

```diff
@@ -0,0 +1,140 @@
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ *   https://www.aklivity.io/aklivity-community-license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package io.aklivity.zilla.runtime.exporter.otlp.internal;
+
+import static io.aklivity.zilla.runtime.exporter.otlp.internal.config.OtlpSignalsConfig.Signals.METRICS;
+
+import java.net.HttpURLConnection;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.time.Duration;
+import java.util.List;
+import java.util.Set;
+import java.util.concurrent.CompletableFuture;
+import java.util.function.LongFunction;
+
+import io.aklivity.zilla.runtime.engine.EngineContext;
+import io.aklivity.zilla.runtime.engine.config.AttributeConfig;
+import io.aklivity.zilla.runtime.engine.config.KindConfig;
+import io.aklivity.zilla.runtime.engine.exporter.ExporterHandler;
+import io.aklivity.zilla.runtime.engine.metrics.Collector;
+import io.aklivity.zilla.runtime.engine.metrics.reader.MetricsReader;
+import io.aklivity.zilla.runtime.exporter.otlp.internal.config.OtlpExporterConfig;
+import io.aklivity.zilla.runtime.exporter.otlp.internal.config.OtlpSignalsConfig;
+import io.aklivity.zilla.runtime.exporter.otlp.internal.serializer.OtlpMetricsSerializer;
+
+public class OltpExporterHandler implements ExporterHandler
+{
+    private static final String HTTP = "http";
+
+    private final OltpConfiguration config;
+    private final EngineContext context;
+    private final Set<OtlpSignalsConfig.Signals> signals;
+    private final String protocol;
+    private final URI metricsUrl;
+    private final long interval;
+    private final Collector collector;
+    private final LongFunction<KindConfig> resolveKind;
+    private final List<AttributeConfig> attributes;
+    private final HttpClient httpClient;
+
+    private OtlpMetricsSerializer serializer;
+    private long nextAttempt;
+    private long lastSuccess;
+    private boolean warningLogged;
+
+    public OltpExporterHandler(
+        OltpConfiguration config,
+        EngineContext context,
+        OtlpExporterConfig exporter,
+        Collector collector,
+        LongFunction<KindConfig> resolveKind,
+        List<AttributeConfig> attributes)
+    {
+        this.config = config;
+        this.context = context;
+        this.metricsUrl = exporter.options().endpoint.resolveMetrics();
+        this.signals = exporter.options().signals.signals;
+        this.protocol = exporter.options().endpoint.protocol;
+        this.interval = Duration.ofSeconds(exporter.options().interval).toMillis();
```

Comment:
> Please add resolveInterval() on exporter to cope with defaults as needed.
done

> Also, suggest supporting generic time period value, such as PT5S via Duration.parse(String) instead of assuming seconds?
My preference is to keep it as seconds. I think the config is easier & more intuitive if we document that you need to specify seconds and people need to write values like `30`, `60`, `120` etc. as opposed to document that you need to specify a duration according to the `ISO-8601` standard and people need to write values like `PT30S`, `PT1M`, `PT2M`. I kept the seconds for now, pls LMK if I should switch over to generic time period values.